packages/server/utils/updateHubspot.ts

```diff
@@ -390,6 +367,12 @@ const updateHubspot = async (event: string, user: IUser, properties: BulkRecord)
     // wait for hubspot to associate the contact with the company, fn must run in 5 seconds
     await sleep(5000)
     await updateHubspotCompany(email, company)
+  } else if (event === 'Account Removed') {
+    const {parabolPayload} = properties
+    if (!parabolPayload) return
+    const {user} = (parabolPayload as unknown) as {[key: string]: any}
```

Comment:
i know this is a bit hacky but wasn't sure if `BulkRecord` type should be changed, since this is a special case we're doing

Reply:
let's just define the shape of `parabolPayload` above with all the fields we expect instead of `any`